.envrc

```diff
@@ -15,6 +15,7 @@ pre-commit install >/dev/null
 dotenv_if_exists .env
 
 if [ ! -d .venv ]; then
+    python3 -m pip install tox
     python3 -m venv .venv
     .venv/bin/pip install -U pip
     .venv/bin/pip install wheel
```

Comment:
hmmm oh I see now -- I usually use `tox --devenv venv` for this usecase.  I don't think this should attempt to globally install `tox` 🤔

Reply:
It complains with:
```
  ValueError: invalid pyproject.toml config: `project.name`.
  configuration error: `project.name` must be pep508-identifier
```

Reply:
ah it wants it to be a `dashed-name` -- maybe set `name = sentry-github-actions-app` ?

Reply:
Thanks! I will look into it another time.